components/crud-web-apps/volumes/backend/apps/rok/pvcviewer.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kubeflow.org/v1alpha1
```

Comment:
@kimwnasptd Is the pvcviewer-controller coming in a separate PR?

Reply:
Not in the immediate future, so we are also OK from a new manifests/registries point of view

Reply:
If this is a Rok exclusive controller (as it stands now), should it be using `kubeflow.org/v1alpha1`? I was already looking into supporting the file viewing feature generically this might cause a clash with the controllers.

Reply:
> If this is a Rok exclusive controller (as it stands now), should it be using kubeflow.org/v1alpha1? I was already looking into supporting the file viewing feature generically which might cause a clash with the controllers.

I'm still debating in my head if such a feature would need to be part of Kubeflow or an external effort where Kubeflow would bundle up different components to provide a solution. In any case this is a really interesting topic we should discuss in the following meetings and draft up some proposals

Reply:
@kimwnasptd I just sent you a few messages in Slack that I think would be the way forward to have this as a generic feature of Kubeflow. I believe the approach I sent you would also allow for the Rok and generic code to be the same, which makes it much easier for future development and maintenance.

Reply:
Let's keep the discussion about this extra functionality separate from this PR. We can discuss and document this effort independently and do any changes necessary in the app afterwards